src/algorithm.jl

```diff
@@ -112,11 +112,33 @@ algorithm `algo` with verbose level `verbose`.
 """
 function sample_scenarios end
 
+function sample_scenarios(sp::AbstractStochasticProgram, n::Int, depth_max = 1000,to::TimerOutput=TimerOutput(), verbose::Int=0)
+    scenarios = []
+    for i = 1:n
```

Comment:
I would tend to follow JuMP guideline for `for` loop: 
https://github.com/JuliaOpt/JuMP.jl/blob/master/docs/src/style.md#for-loops